### PR TITLE
Fix the `SideBarRight` and other improvements

### PR DESCRIPTION
**Pull request by YacineSteeve**

### CODE REVIEW

**Comment by YacineSteeve** [comment]
Current dependencies on/for this PR:
* main
  * **PR #115** <a href="https://app.graphite.dev/github/pr/Frelya/react-site-editor/115" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/Frelya/react-site-editor/115?utm_source=stack-comment).

File: apps/frontend/src/components/Common/ContextMenu/index.ts
Comment (epoundor) [inline apps/frontend/src/components/Common/ContextMenu/index.ts:1]: Why

**Review by epoundor — CHANGES_REQUESTED** [review]

File: apps/frontend/src/utils/remove-non-serializable.ts
[the remark this answers is not in the document]
Comment (YacineSteeve) [reply apps/frontend/src/utils/remove-non-serializable.ts:1]: You're a little genius

File: apps/frontend/src/components/Common/index.ts
[the remark this answers is not in the document]
Comment (YacineSteeve) [reply apps/frontend/src/components/Common/index.ts:1]: These is for gathering components that won't be used in a single feature unit, but across all the app.
For example here the ContextMenu could be added in any place in the app. So can Buttons.

File: apps/frontend/src/components/Common/ContextMenu/index.ts
Comment (YacineSteeve) [reply apps/frontend/src/components/Common/ContextMenu/index.ts:1]: To shorten `ContextMenu/ContextMenu`, and to improve isolation.

As you can see, I created many other index files like that. The purpose of this is to export from each folder ONLY the component that is meant to be used outside of it.

Some examples:

* In the `ContextMenu` folder, it is only the final `ContextMenu` component that would be used somewhere else. The `ContextMenuItem` don't have to.

* In the SideBar folder, only `SideBarLeft` and `SideBarRight` should be exported. The other components are units used to build those two.

File: apps/frontend/src/components/Common/index.ts
[the remark this answers is not in the document]
Comment (epoundor) [reply apps/frontend/src/components/Common/index.ts:1]: Yes I understand the idea.
But I prefer Atomic approach

File: apps/frontend/src/utils/remove-non-serializable.ts
[the remark this answers is not in the document]
Comment (epoundor) [reply apps/frontend/src/utils/remove-non-serializable.ts:1]: Come on
I gonna "rougir"

File: apps/frontend/src/components/Common/index.ts
[the remark this answers is not in the document]
Comment (YacineSteeve) [reply apps/frontend/src/components/Common/index.ts:1]: Let me check about it

**Review by epoundor — APPROVED** [review]

**Pull request merged**